### ISSUE TITLE
rc.40升级了shikiji 到 0.10.0版本之后 “导入代码片段” 功能渲染代码结果 出现问题

### ISSUE THREAD

**Issue by Yjk0010** [vuejs/vitepress#3521]
### Describe the bug

rc.40升级了shikiji 到 0.10.0版本之后 “导入代码片段” 功能渲染代码样式出现问题。

我看到在rc.40版本升级了shikiji的相应版本0.10.0  

但是在其shikiji0.10.0版本对于  “**导入代码片段**”  功能 的代码块 **渲染结果每行内容都增加了一行空行**

渲染结果示意
![image](https://github.com/vuejs/vitepress/assets/27350402/70531526-d5e9-47f4-ac5c-dc313bb3ab6a)

### Reproduction

两个文件书写情况如下

markdown.md
```` md
仅在markdown文件中这样使用会出现

<<<./text.vue

``` vue
这样使用
不会出现
多渲染一行的问题
```

````

text.vue
```` vue

<template>
  <span>这样使用</span>
  <span>会出现</span>
  <span>中间多渲染</span>
  <span>一行</span>
  <span>的问题</span>
</template>


````

源码图片示意
![image](https://github.com/vuejs/vitepress/assets/27350402/46ad9aaf-4db5-4137-82b6-27e49c5c6596)

### Expected behavior

 “导入代码片段”    解析结果应去除空白行。

### System Info

```Text
node 18.19.0
vitepress 1.0.0-rc.40
```

### Validations

- [X] Check if you're on the [latest VitePress version](https://github.com/vuejs/vitepress/releases/latest).
- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.dev).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by brc-dd** [comment]
was fixed in 5811b626576ec4569fa0079d921b8e328d87ca91, should be there in next release.

**Comment by Yjk0010** [comment]
nice
